setup.cfg

```diff
@@ -20,14 +20,14 @@ classifiers =
 packages = df_to_azure
 python_requires = >=3.7
 install_requires =
-    azure-identity==1.6.0
-    azure-mgmt-datafactory==1.1.0
-    azure-mgmt-resource==19.0.0
-    azure-storage-blob==12.8.1
-    pandas==1.3.2
-    pyarrow==5.0.0
-    pyodbc==4.0.31
-    sqlalchemy==1.4.22
+    azure-identity==1.7.1
+    azure-mgmt-datafactory==2.2.0
+    azure-mgmt-resource==20.1.0
+    azure-storage-blob==12.9.0
+    pandas==1.4.0
```

Comment:
We should align this with some other internal packages, since we will get difference in pandas versions between them.

Reply:
Pandas version is equal to azure-batchload, do you seem any other problems? Because now the release candidate is not usable in combination with azure-batchload.

Reply:
There is a mismatch in pandas dependencies, it might work, but it could be unexpected, not what you want. 

Which ones should be in scope for the 'other' packages (azure-batch-load matches with these changes), and why should we do that first (and not after we update df-to-azure in those packages)?